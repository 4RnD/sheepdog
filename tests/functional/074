#!/bin/bash

# Test temporary object lost

. ./common

for i in 0 1 2; do
    _start_sheep $i
done

_wait_for_sheep 3

# start Sheepdog with 3 nodes
_cluster_format -c 1

for i in `seq 0 9`; do
    $DOG vdi create test$i 16M -P
done

for i in 2 1 0; do
    _wait_for_sheep_recovery 0
    _kill_sheep $i
done

_wait_for_sheep_stop

for i in 0 1 2; do
    _start_sheep $i
    _wait_for_sheep $(($i + 1))
    _wait_for_sheep_recovery 0
done

for i in `seq 0 9`; do
    $DOG vdi check test$i
done
