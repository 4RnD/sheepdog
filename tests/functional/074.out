QA output created by 074
using backend plain store
finish check&repair test0
finish check&repair test1
finish check&repair test2
finish check&repair test3
finish check&repair test4
finish check&repair test5
finish check&repair test6
finish check&repair test7
finish check&repair test8
finish check&repair test9
